config/environments/production.rb

```diff
@@ -82,6 +82,30 @@
     config.audit_logger = Logger.new(STDOUT)
   end
 
+  # Whitelist hosts
+  config.hosts << /((ci|qa)\.)?console\.(stage\.)?redhat\.com/
```

Comment:
This needs more coverage for Ephemeral Envs. Currently the hostname that is used there is smth like `front-end-aggregator-ephemeral-NN.apps.c-rh-c-eph.HASH.p1.openshiftapps.com`. But that can change :disappointed: .

Reply:
How important is this to upgrade to rails 6? Could we disable it or just set a wildcard to match any hostname?

Reply:
I don't think that's a wise decision, I would rather go with a more universal wildcard that would also allow something like `*ephemeral*.openshiftapps.com`

Reply:
👍 It would also make sense to expose this setting to set an `ENV` var. ...Is there maybe a hostname ENV var already that could be used?

Reply:
Yeah, thinking about that too.

Reply:
How about: `.+.apps\.c-rh-c-eph\.(\w+)\.p1\.openshiftapps\.com$`? Instead of `(\w+)` we can use concrete cluster hash to make it more secure.